src/main.rs

```diff
@@ -28,9 +30,10 @@ fn print_version() {
 }
 
 fn print_usage(opts: Options) {
-    let usage = concat!("\n\tbuild -- build the cobalt project at the source dir",
+    let usage = concat!("\n\tbuild -- build the cobalt project at the source dir, -i --import will import the site to the gh-pages branch",
```

Comment:
sorry, I meant you already documented it below, I think documenting it in that line is unnecessary :)